### PR TITLE
Add support for new versions of Ubuntu/NodeJS

### PR DESCRIPTION
**Pull request by ColinHebert**

### CODE REVIEW

**Comment by ColinHebert** [comment]
Closes #228

File: manifests/init.pp [hunk 1]
```diff
@@ -31,22 +31,25 @@
     $suffix_error_msg = "Var \$repo_url_suffix with value '${repo_url_suffix}' is not set correctly for ${::operatingsystem} ${::operatingsystemrelease}. See README."
     case $::osfamily {
       'Debian': {
-        # Nodesource repos for Ubuntu lucid and saucy only provide nodejs version 0.10
```
Comment (ColinHebert) [inline manifests/init.pp]: Saucy is gone

File: manifests/init.pp [hunk 2]
```diff
@@ -31,22 +31,25 @@
...
+        # Nodesource repos for Ubuntu lucid only provide nodejs version 0.10
+        if $::operatingsystemrelease =~ /^(10\.04|13.10)$/ {
           validate_re($repo_url_suffix, '^0\.10$', $suffix_error_msg)
         }
         elsif $::operatingsystemrelease =~ /^14\.10$/ {
           validate_re($repo_url_suffix, '^0\.1[02]$', $suffix_error_msg)
         }
-        elsif $::operatingsystemrelease =~ /^1[245]\.04$/ {
-          validate_re($repo_url_suffix, '^(0\.1[02]|[45]\.x)$', $suffix_error_msg)
+        elsif $::operatingsystemrelease =~ /^15\.04)$/ {
```
Comment (bastelfreak) [inline manifests/init.pp]: can you remove the )?

Comment (ColinHebert) [inline manifests/init.pp]: Woops

**Comment by bastelfreak** [comment]
thanks. can you squash that into a single commit?

**Comment by ColinHebert** [comment]
Done

**Comment by bastelfreak** [comment]
thanks \o/

**Comment by ColinHebert** [comment]
Out of curiosity what is the ETA for a release? (I know it barely got merged, but still :P)

**Comment by bastelfreak** [comment]
2minutes

**Comment by ColinHebert** [comment]
Crap that's awesome! thanks!

**Comment by bastelfreak** [comment]
sorry, it took as a bit because of $reasons :cry: but 2.0.1 is now available on the forge.

**Comment by ColinHebert** [comment]
No worries! Thanks for spending the time to release it :)

**Pull request merged**